api/config/config.go

```diff
@@ -0,0 +1,34 @@
+package config
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/joho/godotenv"
+)
+
+var (
+	StringConnectionBD = ""
+	Port = 0
+)
+
+func ConfigEnvironmentVariables() {
+	var erro error
```

Comment:
Aqui está em português:
```suggestion
	var err error
```